src/contour/CMakeLists.txt

```diff
@@ -88,7 +88,7 @@ if(WIN32)
     if (NOT ("${CMAKE_BUILD_TYPE}" STREQUAL "Debug"))
         set_target_properties(contour PROPERTIES
             # Tell MSVC to create a GUI application.
-            WIN32_EXECUTABLE ON
+            #WIN32_EXECUTABLE ON
```

Comment:
this should not have made it in. That however is the reason why I did not see console output in the first place. Even though I was in `CMAKE_BUILD_TYPE=Debug` mode? hmm